packages/fxa-settings/src/components/DropDownAvatarMenu/en-US.ftl

```diff
@@ -0,0 +1,7 @@
+# DropDownAvatarMenu component
+
+drop-down-menu-title = Firefox Account Menu
```

Comment:
Hmm, another complex one. This looks like it could be ` { -product-firefox-account } Menu`, except that one has different casing, "Firefox account".

The right answer here might be to leave the string as-is, but add a localization note above this line similar to what's in the App ftl:

```
# “Account Menu” can be localized, “Firefox” must be treated as a brand.
# This is used to refer to a user's account, e.g. "update your Firefox account ..."
```

Reply:
Does menu need to be capitalized here? The style guide doesn't show this view, https://www.figma.com/file/Nj2WjMOXoJBiPoWvyJ946X/FxA---Manage-Settings-(Copy)?node-id=0%3A982.

I'm in favor of using lower case and going with `{ -product-firefox-account } menu`. Thoughts?

Reply:
Guess this is based on the Monitor menu,
<img width="297" alt="Screen Shot 2021-03-29 at 2 05 19 PM" src="https://user-images.githubusercontent.com/1295288/112879990-e4eb0100-9097-11eb-9939-394a00494f91.png">

Maybe the right thing here is to leave as is?

Reply:
On `Firefox Account` or `Firefox Accounts`,  this is how mozorg handles it (using French as an example): https://pontoon.mozilla.org/fr/mozillaorg/en/brands.ftl/?search=account&string=210503. Whether Account is translated or not, it's locale specific but it is one of the products that can be translated. Also, for mozorg and Firefox for desktop, the brands are put together in a separate file. If we expect more brands to be added, we can consider doing that.

Reply:
> Does menu need to be capitalized here? The style guide doesn't show this view, https://www.figma.com/file/Nj2WjMOXoJBiPoWvyJ946X/FxA---Manage-Settings-(Copy)?node-id=0%3A982.
> 
> I'm in favor of using lower case and going with `{ -product-firefox-account } menu`. Thoughts?

@vbudhram Sure, that sounds good. We've used sentence case in a lot of places already :+1: